rpa-bot/1.0/modules/ROOT/edits/index.adoc

```diff
@@ -0,0 +1,9 @@
+= RPA Bot Overview
+
+RPA Bots are software robots that carry out any RPA processes assigned to them. To execute the processes, RPA Bots require a computer where the applications required for the processes are installed, along with specific configurations for the execution and specific instructions, which are the activities created in RPA Builder for each process step.
```

Comment:
```suggestion
RPA Bots are software robots that carry out any RPA processes assigned to them. To execute the processes, RPA Bots require a computer where the applications required for the processes are installed, along with execution configurations and instructions, which are the activities created in RPA Builder for each process step.
```

Reply:
Confirm globally that bot should be capitalized as RPA Bot. Seems unnecessary, but please confirm